src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractHpiMojo.java

```diff
@@ -1021,6 +1046,28 @@ private String getDevelopersForManifest() throws IOException {
         return buf.toString();
     }
 
+    private void addLicenseAttributesForManifest(Section target) throws ManifestException {
+        final List licenses = project.getLicenses();
+        int licenseCounter = 1;
+        for (Object obj : licenses) {
+            if (obj instanceof License) {
+                License lic = (License)obj;
+                addAttributeIfNotNull(target, "Plugin-Licence-Name-" + licenseCounter, lic.getName());
```

Comment:
British/etc. variant, potentially confusing given the mismatch with the American variant adopted by Maven.

[Background](http://grammarist.com/spelling/licence-license/)